students/choltzman/lesson04/mailroom.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+from textwrap import dedent
+
+# putting variables outside a function like this is ugly, but oh well
+donors = [
+        {
+            "name": "William Gates, III",
+            "gifts": [1234.56, 12345.67]
+        },
+        {
+            "name": "Mark Zuckerberg",
+            "gifts": [9506.31, 8288.91, 1357.02]
+        },
+        {
+            "name": "Jeff Bezos",
+            "gifts": [1234567.89]
+        },
+        {
+            "name": "Paul Allen",
+            "gifts": [1, 2, 3, 4, 5, 6, 7, 8, 9]
+        }
+]
```

Comment:
Note the instructions for the assignment, "
Convert your main donor data structure to be a dict."  You have a list of dictionaries here.  Can you make it one dictionary, with donors as keys?